lib/utils/buildCommandArgs.js

```diff
@@ -56,7 +56,7 @@ export default function buildCommandArgs(
     });
 
   // Use minimist to parse the args, and then build varidiac args and options.
-  const parsedArgs = parseArgs(passedArgs, types);
+  const parsedArgs = parseArgs(passedArgs, types, command._disableTypeCasting === true);
```

Comment:
We could indeed provide access to the `types` array, but I think that's too low level, and it requires users to manually add each flag to this list (and to do it again whenever a flag change). This `disableTypeCasting()` method is higher level and easier to use. It's even possible to disable type casting and then to manually do type casting in the action method.

Also for users who don't want type casting at all anywhere, it makes it easier to disable it by looping on the commands and calling this method.

This is a bit similar to what Vorpal is already doing for flags without arguments, where it adds them automatically to `types.booleans` for convenience.